setup-darwin.sh

```diff
@@ -1,12 +1,13 @@
-# local setup 
+# local setup starting with pkg manager code block 
 if (which brew)
 then
   echo "brew already installed"
 else
   echo "installing brew"
-  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+  /bin/bash -y -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
I think you need an environment variable here, not just an `-y`:

https://github.com/Homebrew/install/?tab=readme-ov-file#install-homebrew-on-macos-or-linux

Reply:
You are right the environment variable is suggested for automation scripts.